pkg/render/logstorage/esmetrics/elasticsearch_metrics.go

```diff
@@ -228,36 +263,14 @@ func (e elasticsearchMetrics) metricsDeployment() *appsv1.Deployment {
 					ServiceAccountName: ElasticsearchMetricsName,
 					InitContainers:     initContainers,
 					Containers: []corev1.Container{
-						relasticsearch.ContainerDecorate(
```

Comment:
Changing how we define the env vars as suggested above means we can also revert this change and keep defining the container inline. Better to touch only the lines of code we need (in this case, the env vars)